### PR TITLE
rename portal

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/about.rst [hunk 1]
```diff
@@ -2,13 +2,13 @@ About
 =====
 
 ----------------------------
-The EBI Metagenomics service
+The MGnify service
 ----------------------------
 
-EBI metagenomics is a freely available hub for the analysis and exploration of :term:`metagenomic`, :term:`metatranscriptomic`, :term:`amplicon` and :term:`assembly` data. The resource provides rich functional and taxonomic analyses of user-submitted sequences, as well as analysis of publicly available metagenomic datasets held within the European Nucleotide Archive (`ENA <https://www.ebi.ac.uk/ena>`_). 
+MGnify is a freely available hub for the analysis and exploration of :term:`metagenomic`, :term:`metatranscriptomic`, :term:`amplicon` and :term:`assembly` data. The resource provides rich functional and taxonomic analyses of user-submitted sequences, as well as analysis of publicly available metagenomic datasets held within the European Nucleotide Archive (`ENA <https://www.ebi.ac.uk/ena>`_).
```
Comment (hudenise) [inline docs/about.rst:8]: should we not indicate that Mgnify was initially EBI Metagenomics?

File: docs/about.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ What does EBI Metagenomics offer
 -------
 Funding
```
Comment (hudenise) [inline docs/about.rst]: The tweeter account is still EBI Metagenomics...

File: docs/dataflow.rst [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
+The graph below summarize the MGnify data flow from submission to :term:`analysis results<analysis result>`:
 
 .. figure:: images/submit_graph_08_web032.png
    :scale: 50 %
 .. https://stackoverflow.com/questions/12297493/why-does-image-scale-not-work-in-restructuredtext-when-generating-html-files
 
 **(1) Submissions are handled by the** `European Nucleotide Archive (ENA) <https://www.ebi.ac.uk/ena/>`_ **and therefore users have to have an ENA** `Webin account <https://www.ebi.ac.uk/ena/submit/sra/>`_.
 
-   In addition, users submitting private data have to provide an expressed agreement that EBI Metagenomics can access their data for analysis, as described under `Submit data <https://www.ebi.ac.uk/metagenomics/submission>`_. Otherwise, we will not be able to access their data. EBI Metagenomics will, of course, handle these data confidentially.
+   In addition, users submitting private data have to provide an expressed agreement that MGnify can access their data for analysis, as described under `Submit data <https://www.ebi.ac.uk/metagenomics/submission>`_. Otherwise, we will not be able to access their data. MGnify will, of course, handle these data confidentially.
```
Comment (hudenise) [inline docs/dataflow.rst]: more a question than a comment: due to GDPR, could we assume that users who gave EBI Metagenomics access agreement have done so too to Mgnify?

File: docs/glossary.rst [hunk 4]
```diff
@@ -20,19 +20,19 @@ Glossary
...
+        An ecological community type. in MGnify, :term:`biomes<biome>` are organised hierarchically going from large types (such as soil, host-associated or aquatic) to more precise types (such as forest soil, skin or coastal) based on the `GOLD classification <https://gold.jgi.doe.gov/distribution#Classification>`_
 
 
     Pipeline
-        A prescribed set of successive steps needed to transform an input (raw reads for EBI Metagenomics) into an output with added information (annotated files with taxonomy and functional assignments for EBI Metagenomics) pipeline tool	a software or script used during the individual step of an analysis pipeline.
+        A prescribed set of successive steps needed to transform an input (raw reads for MGnify) into an output with added information (annotated files with taxonomy and functional assignments for MGnify) pipeline tool	a software or script used during the individual step of an analysis pipeline.
```
Comment (hudenise) [inline docs/glossary.rst]: Mgnify is also accepting assemblies so input should be 'raw reads or contigs'

File: docs/about.rst (hunk 1 above)
Comment (ghost) [reply docs/about.rst:8]: from @almitchell say something like "the resource was previously known as EBI Metagenomics, but was rebranded to reflect its growing remit"

**Pull request merged**